### PR TITLE
Added feat to upload images to Lectures, added progress bar, and image preview. This fixes #101

### LINKED ISSUE #101: feat: allow images to be added to the lectures
* we want to be able to add images to the lecture using and upload button
* ideally the images should be uploaded to firebase

### PR DESCRIPTION
**Pull request by QuintyHH**

### CODE REVIEW

**Review by psyhedeliq — APPROVED** [review]
:1st_place_medal:

File: src/component/lecture/lecture.component.jsx [hunk 1]
```diff
@@ -18,6 +20,38 @@ const Lecture = ({ lectureId }) => {
...
+    if (e.target.files[0]) {
+      setImage(e.target.files[0]);
+    } else {
+      setImage(null);
+    }
+  };
+
+  const handleImageUpload = () => {
+    storage()
+      .ref(`lecture-pictures/${image.name}`)
```
Comment (marianzburlea) [inline src/component/lecture/lecture.component.jsx]: use singular (pictures) -> picture

File: src/component/lecture/lecture.component.jsx [hunk 2]
```diff
@@ -18,6 +20,38 @@ const Lecture = ({ lectureId }) => {
...
+          setImageUploadProgress(
+            (snapshot.bytesTransferred / snapshot.totalBytes) * 100
+          );
+        },
+        error => {
+          console.log(`Woops! ${error.message} while uploading ${image.name}`);
+        },
+        () => {
+          storage()
+            .ref("lecture-pictures")
```
Comment (marianzburlea) [inline src/component/lecture/lecture.component.jsx]: picture**s** -> picture

**Review by marianzburlea — CHANGES_REQUESTED** [review]
remove the "s"es

**Review by marianzburlea — APPROVED** [review]
naspa de 💯

**Pull request merged**